Testsuite/test/run_testsuite_with_cmake

```diff
@@ -66,13 +66,13 @@ wait_for_process()
 
         kill -CONT $pid 2>kill_output 1>/dev/null;   terminated=$?
 # But under CYGWIN the exit status is not to be trusted.
-	if [ $terminated -eq 0 ]; then
-	    if grep -i 'no such process' kill_output; then
-		terminated=1;
-	    fi
-	fi
-	rm -f kill_output
-	if [ $terminated -eq 0 ]
+  if [ $terminated -eq 0 ]; then
+      if grep -i 'no such process' kill_output; then
+    terminated=1;
+      fi
+  fi
+  rm -f kill_output
+  if [ $terminated -eq 0 ]
```

Comment:
Broken indentation.

Even Github is wrong, here, because in the visual diff the origin TABs look like 4 spaces, whereas it should be 8.

Reply:
There are similar indentation issues below...